src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2])
```

Comment:
If you just need to get one element, you only need to use that element's index, e.g. `a[1]`